### PR TITLE
[FIX] website_forum: Shorting posts in descending order by it's votes

### PR DESCRIPTION
**Pull request by jsh-odoo**
Fixing Posts in descending order by it's votes. Fixing the issue of vote update at client side.

Task: https://www.odoo.com/web#id=11288&view_type=form&model=project.task&menu_id=3942&action=327
Pad: https://pad.odoo.com/p/openerp-project.task-W5OW8F4Y4I

### CODE REVIEW

File: addons/website_forum/models/forum.py [hunk 1]
```diff
@@ -172,7 +172,7 @@ def _compute_relevancy(self):
     # vote
     vote_ids = fields.One2many('forum.post.vote', 'post_id', string='Votes')
     user_vote = fields.Integer('My Vote', compute='_get_user_vote')
-    vote_count = fields.Integer('Votes', compute='_get_vote_count', store=True)
+    vote_count = fields.Integer('Votes', compute='_get_vote_count', store=True, default=0)
```
Comment (atp-odoo) [inline addons/website_forum/models/forum.py]: No need to set default=0 here.actually issue is on function _get_vote_count. currently when you didnot get value it's set None, you have to set 0 from function value.

File: addons/website_forum/models/forum.py [hunk 2]
```diff
@@ -172,7 +172,7 @@ def _compute_relevancy(self):
...
```
Comment (tde-banana-odoo) [inline addons/website_forum/models/forum.py]: Ugh ?

Comment (rim-odoo) [inline addons/website_forum/models/forum.py]: @jsh-odoo Make sure that `vote_count` is computed and store at record creation, and that `_get_vote_count` computes correctly `0` in that case.

**Comment by tde-banana-odoo** [comment]
Pushed in saas-6 at commit be31cc299accac150968964274d2d121762994f7 . Thanks !

**Pull request closed without merging**